client/src/components/ProductDetailSubInfo/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { ISubInfo } from '@/interfaces/productDetail.interface';
+import styled from '@emotion/styled';
+import React from 'react';
+
+export interface IProductDetailSubInfo {
+  subInfo: ISubInfo;
+}
+
+const ProductDetailSubInfo = (props: IProductDetailSubInfo): JSX.Element => {
+  const { subInfo } = props;
+  const { title, table } = subInfo;
+
+  const keys = Object.keys(table);
+
+  return (
+    <>
+      <Wrapper>
+        <SubInfoTitle>{title}</SubInfoTitle>
+        <Table>
+          <tbody>
+            {keys.map((key, i) => {
+              return (
+                <Tr key={`information-${i}`}>
+                  <Th>{key}</Th>
+                  <Td>{table[key]}</Td>
+                </Tr>
+              );
+            })}
```

Comment:
```suggestion
            {keys.map((key, i) => (
                <Tr key={`information-${i}`}>
                  <Th>{key}</Th>
                  <Td>{table[key]}</Td>
                </Tr>
              )}
```

리턴을 생략할 수도 있을 것 같아요 ㅎㅎ